database/seed.js

```diff
@@ -0,0 +1,24 @@
+const db = require('./index.js');
+
+const seed = () => {
+  return db.navbarSave(seedGen());
+}
+
+seedGen = function () {
+  var bundleId, countDown;
+  var timerData = [];
+  for (let i = 1; i < 100; i++) {
+    bundleId = i;
+    timer = Number(Math.floor (Math.random() * 13) + 1);
+    countDown = timer;  // countdown in days, may adjust to be more granular later
+    timerData.push ({
+      bundleId : bundleId,
+      countDown : countDown
+    })
+  }
+  return timerData;
+};
```

Comment:
Perhaps I'm misunderstanding the exercise, but the way I read this code, every time the seedGen function is invoked (as it is on line 14 of database/index.js), new data will be generated and then added to the database. Is the intent to stabilize that data at any time? 

Also, every time this runs, it is trying to force bundleIds 1 - 100 into the database. What happens if the database has those bundleIds already and collisions occur?

Reply:
Yeah, I left out the drop table.
Leaving as random at this point until we know if instances should be identical.